frontend/src/metabase/parameters/components/ParameterValueWidget.jsx

```diff
@@ -154,6 +154,8 @@ class ParameterValueWidget extends Component {
               className={cx(S.parameter, className, {
                 [S.selected]: hasValue,
               })}
+              role="button"
+              aria-label={placeholder}
```

Comment:
Fixed so that we only use `placeholder` as a label not also `value` which could change every time users change filters.